media-source/mediasource-is-type-supported.html

```diff
@@ -76,6 +82,17 @@
               'video/mp4;codecs="mp4a.40.2 , avc1.4d001e "',
               'video/mp4;codecs="avc1.4d001e,mp4a.40.5"',
           ], true, 'Test valid MP4 type');
+
+          mediasource_test(function(test, mediaElement, mediaSource)
```

Comment:
This test enforces a non-normative note and doesn't seem appropriate.

I agree with the note in principle, but know of at least one instance in Edge (VP9) where there is only MSE support.  That was done intentionally in part because Edge doesn't currently support Vorbis, and many progressive download uses of VP9 are paired with Vorbis for audio.